scripts/albs-1147.py

```diff
@@ -162,6 +162,56 @@ async def main(write=False):
         if write:
             await session.commit()
 
+    repos_to_publicate = set()
+    pulp = PulpClient(settings.pulp_host, settings.pulp_user, settings.pulp_password)
+
+    with get_pulp_db() as pulp_db, pulp_db.begin():
+        repos = pulp_db.execute(select(CoreRepository)).scalars().all()
+        for repo in repos:
+            repo_href = f"/pulp/api/v3/repositories/rpm/rpm/{repo.pulp_id}/"
+
+            repo_subq = (
+                select(CoreRepository.pulp_id).where(
+                    CoreRepository.name == repo.name
+                )
+            )
+
+            repocontent_subq = (
+                select(CoreRepositoryContent.content_id).where(
+                    CoreRepositoryContent.repository_id.in_(repo_subq)
+                )
+            )
+
+            content_subq = (
+                select(CoreContent.pulp_id).where(
+                    CoreContent.pulp_type == 'rpm.advisory',
+                    CoreContent.pulp_id.in_(repocontent_subq)
+                )
+            )
+
+            updaterecord_subq = (
+                select(UpdateRecord).where(
+                    and_(
+                        UpdateRecord.content_ptr_id.in_(content_subq),
+                        UpdateRecord.content_ptr_id.in_(
+                            {v['content_ptr_id'] for k, v in affected_records.items()})
+                    )
+                )
+            )
+
+            records = pulp_db.execute(updaterecord_subq).scalars().all()
+
+            for record in records:
```

Comment:
I see selects, but where the update itself?

Reply:
This is preparation only. The update is down below. 
`pulp.create_rpm_publication(repo_href)`